apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/carin/C4BBPractitionerIdentifierType.java

```diff
@@ -27,6 +28,8 @@ public String toCode() {
         return "UPIN";
       case PIN:
         return "PIN";
+      case TAX:
+        return "PIN";
```

Comment:
cut-past error? Should be "TAX" shouldn't it?

Reply:
done